controllers/status.go

```diff
@@ -191,32 +187,34 @@ func calculateStatus(
 
 	// Copy conditions one by one to not affect original object
 	conditions := hub.Status.HubConditions
-	for i := range conditions {
-		status.HubConditions = append(status.HubConditions, conditions[i])
```

Comment:
If they're supposed to be in a certain order, do we want to change this?

Reply:
I don't believe they are supposed to be within a certain order; however, I can revert this change if needed.